compat/src/test/scala/test/scala/collection/Playground.scala

```diff
@@ -0,0 +1,12 @@
+package test.scala.collection
+
+import org.junit.Test
+import org.junit.Assert._
+import scala.collection.compat._
+
+class Playground {
+  @Test
+  def t(): Unit = {
+
+  }
```

Comment:
What’s the purpose of this test?

Reply:
It's an empty test you can use as a sandbox. I keep finding myself to add it back locally, It's useful to have it here.

Reply:
I find this a bit weird. Can’t you just have it locally if you only use it during development?

Reply:
Can we keep it, it would save me some time when I want to test something quick or if I need to create another unit test.

Reply:
I don’t understand why you can’t keep it locally only.